module/input_handler.py

```diff
@@ -1,56 +1,43 @@
-from module.command_node import NodeType
-from module.frame_functions import cast_and_send_ui_frame
+from module.command_node import Node, Command
 
 
 class input_handler:
     def __init__(self, cli_controller):
         self.cli_controller = cli_controller
 
     @staticmethod
-    def print_command_not_found(command: str) -> None:
+    def _print_command_not_found(command: str) -> None:
         """
         Prints command not found text with the command parameter.
         """
-        print("\tCommand '{}' not found, type 'help' for possible commands.".format(command))
+        print(
+            "\tCommand '{}' not found, type 'help' for possible commands.".format(
+                command
+            )
+        )
 
     @staticmethod
-    def check_amount_parameters(parameters: list, required_amount: int):
+    def _check_amount_parameters(parameters: list, required_amount: int) -> bool:
         """
         compares the length of the parameters list to the required amount,
         returns true when the length of the parameters is equal to the required amount
         prints to user if not enough or too many parameters
         """
         if len(parameters) < required_amount or len(parameters) > required_amount:
-            print("\tExpected {} parameters, got {}.".format(required_amount, len(parameters)))
+            print(
```

Comment:
Why does this check have to print? In my option a check only checks something and somewhere else the handling is handled. Like printing an error.